tests/phpunit/bootstrap.php

```diff
@@ -17,7 +17,7 @@ function _manually_load_plugin() {
 
 require $_tests_dir . '/includes/bootstrap.php';
 
-$wc_tests_framework_base_dir = dirname( dirname( dirname( dirname( __FILE__ ) ) ) ) . '/woocommerce/tests/framework/';
+$wc_tests_framework_base_dir = dirname( dirname( dirname( dirname( __FILE__ ) ) ) ) . '/woocommerce/tests/legacy/framework/';
```

Comment:
Good catch! When I needed to run tests for the first time last week, I was doing some poor work-arounds. Now I don't need to.